simple-weather/simple-weather/Base.lproj/Main.storyboard

```diff
@@ -95,6 +96,9 @@
                     <connections>
                         <outlet property="dateLabel" destination="i1j-a9-yrl" id="i7e-Tg-Jt8"/>
                         <outlet property="locationLabel" destination="dNR-jB-Zbv" id="wbg-6O-dGN"/>
+                        <outlet property="tempLabel" destination="hql-5C-zOI" id="egu-WK-edb"/>
```

Comment:
`Networking` branch has UI changes, branch should have only relevant changes. Networking is getting data, displaying data is UI, therefore, `networking` branch should get data, new branch, for example, `displaying-data` should have had these outlets, and using data to populate labels